docs/quickstart/swagger_generation.rst

```diff
@@ -169,35 +169,69 @@ Notice that since we've started to customize the swagger generator, we should sp
 Authenticators
 ^^^^^^^^^^^^^^
 
-We also need to tell the generator how to represent custom Authenticators as Swagger.
+We also need to tell the generator how to represent custom Authenticators as Swagger. Note that this changed somewhat in Flask-rebar 2.0:
```

Comment:
How about using similar syntax to the python docs where we add a note denoting if a feature had a breaking changed in a major version; could be a bit more concise?
* changed in 2.0
* new in 2.0

Reply:
I like it - will adjust